akka-remote/src/main/resources/reference.conf

```diff
@@ -671,8 +669,8 @@ akka {
         # the JCE Unlimited Strength Jurisdiction Policy Files to use AES 256.
         # More info here:
         # https://www.oracle.com/java/technologies/javase-jce-all-downloads.html
-        enabled-algorithms = ["TLS_RSA_WITH_AES_128_CBC_SHA",
-        "TLS_DHE_RSA_WITH_AES_256_GCM_SHA384"]
+        enabled-algorithms = ["TLS_DHE_RSA_WITH_AES_256_GCM_SHA384",
```

Comment:
Sounds reasonable, this one seems similar to the one recommended in TLS 1.3 (`TLS_AES_256_GCM_SHA384`).

(Though, if people would want to use TLS 1.3, you would need a different set of algorithms, as those are not compatible between 1.2 and 1.3)

If you can you should probably go with TLS 1.3 but since that's not available in JDK 8, we probably cannot recommend it here.

For the particular case of TLS in akka-remote, is there a reason to provide a list here? As noted above, in the usual case, you should be in control of both the client and the server, so you should be able to choose freely and only provide more than one in case of a rolling upgrade.